examples/cfd/darcy_nested_fnos/utils.py

```diff
@@ -224,8 +217,9 @@ def compare(
             fig.colorbar(im[ii], ax=ax[ii], location="bottom", fraction=0.046, pad=0.04)
             ax[ii].set_title(self.headers[ii])
 
-        mlflow.log_figure(fig, f"val_step_{step}.png")
-        fig.savefig(os.path.join(self.out_dir, f"validation_step_{step}.png"))
+        LaunchLogger("valid").log_figure(

```

Comment:
You should pass the logger in as a parameter to this function not instantiate.

This will reset the epoch number so when it logs in console, it will read epoch 1 for all validation.

Reply:
done